pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -598,6 +600,18 @@ self: super: {
     '';
   });
 
+  mason-nvim = super.mason-nvim.overrideAttrs (old: {
+    dependencies = with self; [ git curl gnutar gzip ];
```

Comment:
I don't think this will work as expected

Reply:
I think we can safely remove this override since they are common tools, reading through the source code I think patching might be hard to do, and there is not much else we can do at the moment without https://github.com/NixOS/nixpkgs/pull/208029